component/core/src/test/java/org/exoplatform/social/core/jpa/storage/dao/LabelDAOTest.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * This file is part of the Meeds project (https://meeds.io/).
+ *
+ * Copyright (C) 2023 Meeds Association contact@meeds.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package org.exoplatform.social.core.jpa.storage.dao;
+import org.exoplatform.social.core.jpa.storage.dao.jpa.LabelDAO;
+import org.exoplatform.social.core.jpa.storage.entity.LabelEntity;
+import org.exoplatform.social.core.jpa.test.BaseCoreTest;
+
+import java.util.List;
+
+public class LabelDAOTest extends BaseCoreTest {
+  private LabelDAO labelDAO;
+  private String objectId = "1";
+  private String objectType = "testObject";
+
+
+  @Override
+  public void setUp() throws Exception {
+    super.setUp();
+    labelDAO = getService(LabelDAO.class);
+  }
+
+  @Override
+  public void tearDown() throws Exception {
+    labelDAO.deleteAll();
+    super.tearDown();
+  }
+
+  public void testCreateLabel() throws Exception {
+    LabelEntity labelEntity = createLabel("test label fr", "fr");
+
+    labelDAO.create(labelEntity);
+
+    end();
+    begin();
+
+    LabelEntity result = labelDAO.find(labelEntity.getId());
+    assertNotNull(result);
+    assertEquals(result.getId(), labelEntity);
+  }
+
+  public void testFindLabelByObjectTypeAndObjectIdAndLang() throws Exception {
+    LabelEntity labelEntityFr = createLabel("test label fr", "fr");
+
+    labelEntityFr = labelDAO.create(labelEntityFr);
+
+    LabelEntity labelEntityEn = createLabel("test label en", "en");
+
+    labelEntityEn = labelDAO.create(labelEntityEn);
+
+
+    end();
+    begin();
```

Comment:
please apply for all in this file

```suggestion
        restartTransaction();
```

Reply:
Done